reactive/kubernetes_control_plane.py

```diff
@@ -1398,8 +1398,10 @@ def get_dns_info():
             dns_ip = kubernetes_control_plane.get_dns_ip()
         except CalledProcessError:
             hookenv.log("DNS addon service not ready yet")
+        if dns_ip:
+            return True, dns_ip, 53, dns_domain
+        else:
             return False, None, None, None
```

Comment:
Random nit:  this method could REALLY use some return typing 

`def get_dns_info() -> Tuple[bool, Optional[str], Optional[int], Optional[str]]: `

```suggestion
        if dns_ip:
            return True, dns_ip, 53, dns_domain
        return False, None, None, None
```